client/react/frontpage/components/SocialPanel.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import './SocialPanel.scss';
+
+// update social media links here
+// img is the base name of the Font Awesome icon
+const socialMediaLinks = [
+  {
+    img: 'facebook',
+    link: 'https://www.facebook.com/uclaradio',
+  },
+  {
+    img: 'twitter',
+    link: 'https://twitter.com/UCLAradio',
+  },
+  {
+    img: 'instagram',
+    link: 'https://www.instagram.com/uclaradio/',
+  },
+  {
+    img: 'snapchat-ghost',
+    link: 'https://www.snapchat.com/add/uclaradio',
+  },
+];
+
+const SocialPanel = React.createClass({
+  convertToFaName(img, size) {
+    return 'fa fa-' + img + ' fa-' + size;
+  },
+  render() {
+    const self = this;
+    return (
+      <div>
+        <p className="infoHeader">Social Media</p>
```

Comment:
Hmm I'm not a big fan of the header saying "Social Media"—I feel like it's a little redundant. However, not having the banner also looks weird...

Lemme dig up some old mockups and see if I can suggest a better designed solution.

Reply:
Removed the header for now